tendrl/commons/event.py

```diff
@@ -0,0 +1,10 @@
+import socket
+import json
+
+
+class Event(object):
+    def __init__(self, message, host, port):
+        s = socket.socket()
+        s.connect((host, port))
+        s.sendall(json.dumps(message.__dict__))
+        s.shutdown(socket.SHUT_RDWR)
```

Comment:
Class initialisation shouldn't do any action. It should just setup the object's initial state. Another method should be invoked to do the actual action. In this case, only the variables (including the socket object, but not the actual connection) should be initialised.

The actual writing to the socket should happen in a `write` method. This `write` method should open the socket connection, write the data and then close the connection.